### PR TITLE
Make User Guide more reader friendly

### PR DESCRIPTION
**Pull request by jiale-c**
Update the sections on edit and summarise so that the explanation is more layman and easy to read.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -117,70 +112,98 @@ Examples of usage:
...
 * You are able to add optional tags with no restrictions using the `t/` prefix.
 
+### Editing an existing contact
+Edit a contact at a specific index.
+
+Format: `edit INDEX n/NAME ...`
+* Edits an existing student's information found in Tracey. The student is referred by an index, displayed alongside him in the student list.
+* To edit a student, you can use the command `edit` on the student's `INDEX` with the optional fields of information to be changed.
+  The index **must be a positive integer** 1, 2, 3, …
+* You have to provide at least one of the optional fields of information.
```
Comment (DannyDakota) [inline docs/UserGuide.md:127]: What does this line mean? which are the optional fields? if we are going to include this line, I think we should spell out clearly to the reader which are the optional and compulsory fields

File: docs/UserGuide.md [hunk 2]
```diff
@@ -117,70 +112,98 @@ Examples of usage:
...
+* Existing information of the student will be updated with the newly inputted values.
+* The original value to be edited must not be the same as the corresponding field value of the student.
```
Comment (DannyDakota) [inline docs/UserGuide.md:129]: I think this might be a bit too technical 👍

**Review by DannyDakota — APPROVED** [review]
LGTM! just some minor changes to the sentence structure used

File: docs/UserGuide.md (hunk 1 above)
Comment (jiale-c) [reply docs/UserGuide.md]: Thank you. I have made the description simpler.

**Pull request merged**